src/main/java/alex/Alex.java

```diff
@@ -0,0 +1,90 @@
+package alex;
+
+import alex.storage.Storage;
+import alex.tasklist.TaskList;
+import alex.ui.Ui;
+
+import java.util.Scanner;
+
+/**
+ * Represents the chatbot that is able to respond to commands given by the user.
+ * An Alex object corresponds to a chatbot represented by a string referring to the
+ * data file where the bot will load the list of tasks from.
+ */
+public class Alex {
+    private String filePath;
+    private Storage storage;
+    private TaskList tasks;
+    private Ui ui;
+    public static final String LINE = Ui.LINE;
+
+
+    public Alex(String filePath) {
+        this.filePath = filePath;
```

Comment:
Could look at java Paths class instead of using Strings when dealing with file paths to avoid environment related errors.